File: StraviaTec-Web/Models/EventoGrupo.cs
using System;
using System.Collections.Generic;

namespace StraviaTec_Web.Models
{
    public partial class EventoGrupo
    {
        public int Id { get; set; }
        public int IdEvento { get; set; }
        public int IdGrupo { get; set; }

        public virtual Evento IdEventoNavigation { get; set; }
        public virtual Grupo IdGrupoNavigation { get; set; }
    }
}

File: StraviaTec-Web/ClientApp/src/app/models/eventoGrupo.ts
export class EventoGrupo {
    id: number;
    idEvento: number;
    idGrupo: number;
    nombreGrupo: string;

    constructor(
        id: number,
        idEvento: number,
        idGrupo: number) {
            this.id = id;
            this.idEvento = idEvento;
            this.idGrupo = idGrupo;
        }
}